src/main/resources/application.properties

```diff
@@ -15,9 +15,9 @@ spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 
 ## PostgreSQL
 spring.datasource.platform=postgres
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.url=jdbc:postgresql://places-postgresdb.postgres.database.azure.com:5432/postgres?user=climatetreeops@places-postgresdb&password=P@ssw0rd@123!&sslmode=require
```

Comment:
You can remove the user name and password from this string since they are defined below

Reply:
its part of the PR description that this will be addressed in a later PR. I can remove it right now. but devops has to ensure they are going to set those 2 variables in the server where they will deploy this microservice app.

Reply:
You could even create and env variable for the Database name

Reply:
Also, is this the username ? 
![image](https://user-images.githubusercontent.com/46201396/74091579-08637e80-4a87-11ea-9701-502c3c3b076e.png)

Reply:
Ah ok, gotcha. But, yes that is the username